examples/RequestedBBRCProblem.ipynb

```diff
@@ -0,0 +1,101 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test de la classe RequestedBBRCProblem\n",
+    "\n",
+    "L'objectif de cet exemple est de présenter l'utilisation de la classe RequestedBBRCProblem"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openturns as ot\n",
+    "import otbenchmark as otb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": "RP8\n"
+    }
+   ],
+   "source": [
+    "problem = otb.RequestedBBRCProblem(\"testuser\", \"testpass\", -1, 1)\n",
+    "print(problem.name)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "event = problem.getEvent()\n",
+    "g = event.getFunction()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": "0.0007888456943755395"
+     },
+     "metadata": {},
+     "execution_count": 4
+    }
+   ],
+   "source": [
+    "problem.getProbability()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the Monte-Carlo algorithm\n",
+    "algoProb = ot.ProbabilitySimulationAlgorithm(event)\n",
+    "algoProb.setMaximumOuterSampling(10)\n",
+    "algoProb.setMaximumCoefficientOfVariation(0.01)\n",
+    "algoProb.run()"
```

Comment:
Please insert a comment that explains that your intent in selecting a low value of the maximumOuterSampling : it is chosen to make a fast example, but a more realistic parameter value (e.g. 1000) should be chosen in practice. This will clarify your motivations in the example. Please insert the code to print the estimated probability, because it is the concrete goal of the example but add a comment that explains that, because of the low value of the maximumOuterSampling, the estimate is not very accurate.